docs/pages/guides/in-app-purchases.mdx

```diff
@@ -50,12 +50,15 @@ The following libraries provide robust support for in-app purchase functionality
 />
 
 <BoxLink
-  title={
-    <>
-      <CODE>react-native-iap</CODE>
-    </>
-  }
-  description="A React Native library providing an interface to the client-side native in-app purchase API's for Android (both Play Store and Amazon) and iOS."
-  href="https://github.com/dooboolab-community/react-native-iap"
+  title={<><CODE>react-native-iap</CODE> (Deprecated)</>}
+  description="A legacy React Native library for in-app purchases. Superseded by expo-iap for Expo users."
+  href="https://github.com/hyochan/react-native-iap"
+  Icon={GithubIcon}
+/>
+
+<BoxLink
+  title={<><CODE>expo-iap</CODE></>}
+  description="The official Expo module for in-app purchases, supporting managed workflows with dev-client."
```

Comment:
```suggestion
  description="A React Native library for in-app purchases that works with development builds."
```

- "managed workflows" is a deprecated term. We prefer using development builds to indicate that a library contains native API integration, and the developer needs to create a development build in order to use the library. I'd like to avoid it here as well.
- The "official" means that a library is maintained by the Expo team, which is not true in this case. Let's remove it.